src/js/themes/base.d.ts

```diff
@@ -0,0 +1,762 @@
+import { DeepReadonly, ColorType, OpacityType, BackgroundType } from '../utils'
+
+export declare const base: DeepReadonly<ThemeType>;
+export declare const generate: (baseSpacing: number, scale: number) => DeepReadonly<ThemeType>;
+
+declare const colors: {
+  active: ColorType;
+  black: ColorType;
+  border: ColorType;
+  brand: ColorType;
+  control: ColorType;
+  focus: ColorType;
+  placeholder: ColorType;
+  selected: ColorType;
+  text: ColorType;
+  icon: ColorType;
+  white: ColorType;
+};
+  
+type Colors = typeof colors & {
```

Comment:
all these colors should also support ColorType structure, https://codesandbox.io/s/accent-1-i19hp